CHANGELOG.md

```diff
@@ -10,6 +13,7 @@
 ### Features
 
 - New source generator allows Sentry to see true build variables like PublishAot and PublishTrimmed to properly adapt checks in the Sentry SDK ([#4101](https://github.com/getsentry/sentry-dotnet/pull/4101))
+- Auto breadcrumbs now include all .NET MAUI gesture recognizer events ([#4124](https://github.com/getsentry/sentry-dotnet/pull/4124))
```

Comment:
This is from another PR right?

Reply:
Yes.  I needed the work for the visual runner here